lib/liquid/tags/snippet.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Liquid
+  # @liquid_type tag
+  # @liquid_category theme
```

Comment:
I believe we could use `variable` to be consistent with the `capture` tag (also thinking it created a `SnippetDrop`)

```suggestion
  # @liquid_category variable
```